src/lib/translators/mongo.ts

```diff
@@ -53,6 +53,21 @@ function columnMap(colnames: string[]) {
   return _.fromPairs(colnames.map(col => [col, $$(col)]));
 }
 
+/**
+ * Generate a mongo [user variable](https://docs.mongodb.com/manual/reference/aggregation-variables/#user-variables)
+ * valid identifier from a column name.
+ *
+ * @param colname
+ */
+function columnToUserVariable(colname: string) {
```

Comment:
```suggestion
function columnToUserVariable(colname: string): string {
```
doesn't cost anything